packages/webcomponents/src/api/services/terminal.service.ts

```diff
@@ -22,9 +21,8 @@ export class TerminalService implements ITerminalService {
     params: any,
     apiOrigin?: string
   ): Promise<IApiResponseCollection<ITerminal[]>> {
-    
     if (!apiOrigin) {
-      apiOrigin = config.proxyApiOrigin;
+      apiOrigin = PROXY_API_ORIGIN;
```

Comment:
Another one we should be able to shorten by assigning this in the interface I believe